tools/bridge/bridge/transfer_recorder.py

```diff
@@ -87,12 +91,18 @@ def pull_transfers_to_confirm(self) -> List[AttributeDict]:
             confirmation_tasks = []
 
         self.clear_transfers()
+        sort_events(confirmation_tasks)
         return confirmation_tasks
 
     def apply_proper_event(self, event: AttributeDict) -> None:
         event_name = event.event
 
         if event_name == TRANSFER_EVENT_NAME:
+            if event.args.value == 0 or is_same_address(
+                event.args["from"], ZERO_ADDRESS
+            ):
```

Comment:
Do we actually want the balance check? It would be a nice way to test if the bridge is working without having tokens. And for attackers it doesn't really make a difference if they send 0 or 1 wei.

the "from" address will never be zero, so maybe an `assert` is more appropriate.

Reply:
HomeBridge.sol will reject these transfers (I accidentally created one with zero amount). So, yes, we want the balance check.
Our TrustlinesNetworkToken.sol has an emit Transfer with the zero address as sender in it's mint function. So, I would also like to filter those out.